yb-voyager/src/reporter/stats/streamImportReporter.go

```diff
@@ -41,20 +43,25 @@ type StreamImportStatsReporter struct {
 	remainingEvents        int64
 	estimatedTimeToCatchUp time.Duration
 	uitable                *uilive.Writer
+	metaDB                 *metadb.MetaDB
 }
 
 func NewStreamImportStatsReporter() *StreamImportStatsReporter {
 	return &StreamImportStatsReporter{}
 }
 
-func (s *StreamImportStatsReporter) Init(tdb tgtdb.TargetDB, migrationUUID uuid.UUID) error {
+func (s *StreamImportStatsReporter) Init(tdb tgtdb.TargetDB, migrationUUID uuid.UUID, exportDir string) error {
 	s.migrationUUID = migrationUUID
 	numInserts, numUpdates, numDeletes, err := tdb.GetTotalNumOfEventsImportedByType(migrationUUID)
 	s.totalEventsImported = numInserts + numUpdates + numDeletes
 	if err != nil {
 		return fmt.Errorf("failed to fetch import stats meta info from target : %w", err)
 	}
 	s.startTime = time.Now()
+	s.metaDB, err = metadb.NewMetaDB(exportDir)
```

Comment:
Let's ensure that there is only ONE instance of `MetaDB` in the whole voyager run.

If possible take `*MetaDB` as an argument to `Init()` -- instead of the `exportDir`.